public/ajax/callbacks.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable import/extensions */
+import eventBus from '../scripts/eventBus.js';
+import ajax from './script.js';
+
+const backendAddress = 'https://ozonback.herokuapp.com';
+
+export const signin = (data) => {
+  console.log(data);
+  ajax.post({
+    url: `${backendAddress}/login`,
+    body: data
+  })
+    .then(({ responseText }) => eventBus.emit('signin success', responseText))
```

Comment:
'signin success' -- вынесите в константы названия событий, можно в объект

```suggestion
    .then(({ responseText }) => eventBus.emit(events.signinSuccess, responseText))
```